include/lsst/afw/image/ExposureInfo.h

```diff
@@ -197,6 +197,12 @@ class ExposureInfo final {
     /// Set the exposure's transmission curve.
     void setTransmissionCurve(std::shared_ptr<TransmissionCurve const> tc) { _transmissionCurve = tc; }
 
+    /// Get the version of FITS serialization that this ExposureInfo understands.
+    static const int getFitsSerializationVersion() { return _fitsSerializationVersion; }
+
+    /// Get the version of FITS serialization version info name
+    static const std::string getFitsSerializationVersionName() { return _fitsSerializationVersionName; }
```

Comment:
This should return `std::string const &` (or just `std::string`) and the one above should just return `int`; there's no reason to add `const` to a by-value return type.

I would also recommend reimplementing these to use function-local static variables, instead of a static class data member.  In other words, you'd declare but not define these functions here, and then in `ExposureInfo.cc`, you'd have e.g.:
```
std::string const & ExposureInfo::getFitsSerializationVersionName() {
    static std::string const versionName("EXPINFO_V");
    return versionName;
}
```

What you've done here is not incorrect, and in fact it's very unlikely to cause problems.  But it leaves the door slightly open for one of the trickier and more obscure ways to get C++ to segfault: the [static initialization order problem](https://isocpp.org/wiki/faq/ctors#static-init-order), while what I've recommended instead closes that door.  (You may also note that link advocates a slightly different solution, which closes the door even more securely but makes `valgrind` generate spurious warnings; I'm okay with either but personally prefer to keep valgrind output clean in this case.)